test/extended/cmd.sh

```diff
@@ -54,88 +54,89 @@ oadm policy add-role-to-group view system:authenticated -n default
 
 install_registry
 wait_for_registry
-DOCKER_REGISTRY=`oc get service/docker-registry -n default -o jsonpath='{.spec.clusterIP}:{.spec.ports[0].port}'`
+docker_registry="$( oc get service/docker-registry -n default -o jsonpath='{.spec.clusterIP}:{.spec.ports[0].port}' )"
```

Comment:
The guide says, "Constants and anything exported to the environment should be capitalized", and it uses lower-case in examples with local variables.  I agree that using upper-case for globals and lower-case for locals is good style.